flipperdevices/flipperzero-firmware
# fbt: add `fap_deploy` target to build and copy all .fap apps to flipper

# What's new

This adds a new `./fbt` target called `fap_deploy`. This builds and uploads all discovered FAPs to the connected Flipper via USB. Similar to how `./fbt launch_app` builds and uploads a single FAP over USB, `fap_deploy` builds and copies all FAPs (no apps are launched).

I wrote this for easier (re)building and uploading of all FAPs when the Flipper firmware is updated and the ABI version changes.

# Verification 

* Remove all existing FAP build artifacts: `rm -rf build/f7-firmware-D/.extapps/`
* Connect Flipper via USB
* Run `./fbt fap_deploy`
* Observe FAPs are built and copied to the connected Flipper

# Checklist (For Reviewer)
